libs/util/shared/src/lib/elasticsearch/elasticsearch.service.ts

```diff
@@ -48,6 +49,45 @@ export class ElasticsearchService {
     }
   }
 
+  getRelatedRecordPayload(
+    title: string,
+    size: number = 6,
+    _source = ES_SOURCE_SUMMARY
+  ): EsSearchParams {
+    return {
+      query: {
+        bool: {
+          must: [
+            {
+              more_like_this: {
```

Comment:
I think this is not in line with customer requirements: the idea for related records was to show records which have an association with the current one: sibling, parent, child, service etc.

Reply:
.... the mockup doesn't tell anything about what kind of relation it is.
doesn't tell either what happens if more than 3
we can keep the code cause it's going to be usefull anyway, if they want the `gn-related` directive, then I won't do it personally, no more functional dev for this contract. And I don't like the semi-dozens of requests sent to fetch the `related` in gn3